### ISSUE TITLE
Making U/UF6 ratio a member variable 

### ISSUE THREAD

**Issue by stompsjo** [stompsjo/mbmore#3]
https://github.com/stompsjo/mbmore/blob/9eca5e922b5a98d5d8d9f1811182f2f6607b0a4b/src/CentrifugeConfig.cc#L100

This ratio is used in multiple areas of `CentrifugeConfig` and `StageConfig` to correct for separation work differences when treating U vs. UF6. Can this ratio be brought out of the calculations themselves and added as a member variable to their respective classes?

The only issue I forsee is that `CentrifugeConfig.M` can be changed, which may cause mismatch between the mass and the ratio if the ratio is not re-initialized. Right?

**Comment by maxschalz** [comment]
> The only issue I forsee is that `CentrifugeConfig.M` can be changed, which may cause mismatch between the mass and the ratio if the ratio is not re-initialized. Right?

I agree, this could cause unnecessary confusion if one's not aware of it. Two possibilities to avoid this:
- make `CentrifugeConfig.M` a `const` variable which effectively makes the ratio constant as well (or you can set it explicitly to `const`). The downside with this is that `M` obviously cannot be changed (on the other hand: why should one change it?)
- implement `ratio_UF6_U` as a function (or inline function) instead of a variable?
